src/script/script_instance.cpp

```diff
@@ -126,12 +126,31 @@ bool ScriptInstance::LoadCompatibilityScripts(const std::string &api_version, Su
 
 		if (this->engine->LoadScript(buf)) return true;
 
-		ScriptLog::Error("Failed to load API compatibility script");
+		ScriptLog::Error(fmt::format("Failed to load API compatibility script for {}", api_version));
 		Debug(script, 0, "Error compiling / running API compatibility script: {}", buf);
 		return false;
 	}
 
-	ScriptLog::Warning("API compatibility script not found");
+	ScriptLog::Warning(fmt::format("API compatibility script for {} not found", api_version));
+	return true;
+}
+
+bool ScriptInstance::LoadCompatibilityScripts(Subdirectory dir, const std::list<std::string> &versions)
```

Comment:
```suggestion
bool ScriptInstance::LoadCompatibilityScripts(Subdirectory dir, std::span<std::string> versions)
```
If that doesn't work:
```suggestion
bool ScriptInstance::LoadCompatibilityScripts(Subdirectory dir, const std::initializer_list<std::string> &versions)
```